cli.js

```diff
@@ -9,46 +9,77 @@ const express = require('express')
 const graphqlHTTP = require('express-graphql')
 const OasGraph = require('./lib/index.js')
 const path = require('path')
+const http = require('http')
+const fs = require('fs');
 
-const app = express()
-const [,, ... args] = process.argv
-
-// Check for arguments
-if (args.length > 0) {
-  let filePath = args[0]
-
-  // Convert path to absolute path
-  // Otherwise, would need to have separate cases for relative or absolute paths
-  filePath = path.resolve(filePath)
-
-  try {
-    let oas = require(filePath)
-
-    // Create GraphQL interface
-    OasGraph.createGraphQlSchema(oas)
-      .then(({schema, report}) => {
-        console.log(JSON.stringify(report, null, 2))
-        app.use('/graphql', graphqlHTTP({
-          schema: schema,
-          graphiql: true
-        }))
-
-        app.listen(3001, () => {
-          console.log('GraphQL accessible at: http://localhost:3001/graphql')
-        })
-      })
-      .catch(err => {
-        console.log(err)
-      })
-  } catch (error) {
-    if (error.code === 'MODULE_NOT_FOUND') {
-      console.log(`The file path "${filePath}" is invalid.`)
+const app = express();
+
+if (process.argv.length <= 2) {
+     console.log("Usage: " + __filename + " <OAS JSON file path or remote url> <port number>");
+     process.exit(-1);
+}
+
+let filePath = process.argv[2];
+let portNumber = process.argv[3] ? process.argv[3] : 3001;
+
+// check if the file exists 
+if (fs.existsSync(path.resolve(filePath))) {
+      let oas = JSON.stringify(require(path.resolve(filePath)));
+      startGraphQLServer(oas, portNumber);
+    
+} else { // falls back to a remote location
+    if (filePath.includes('http')) {
```

Comment:
Better use something like `/^https?/.test(filePath)` here, because it might just be that `'http'` appears in the middle of a filepath.